scilpy/io/hdf5.py

```diff
@@ -112,17 +121,20 @@ def reconstruct_sft_from_hdf5(hdf5_handle, group_keys, space=Space.VOX,
 
     # 3) Format as SFT
     if merge_groups:
-        if len(streamlines) == 0:
-            logging.info("Empty SFT not reconstructed from hdf5.")
-            return None
+        if len(streamlines) == 0 and not allow_empty:
+            raise ValueError("Empty SFT cannot reconstructed from hdf5. "
```

Comment:
"Cannot load an empty SFT from HDF5. Set `allow_empty` to True if you want to force it."